images/nginx/rootfs/build.sh

```diff
@@ -168,8 +168,8 @@ get_src 2a69815e4ae01aa8b170941a8e1a10b6f6a9aab699dee485d58f021dd933829a \
 get_src f74a0821b079ea1fd63dd8659064356fc3f421ff4b35c17877140d2b2841cc3b \
         "https://github.com/openresty/luajit2/archive/v$LUAJIT_VERSION.tar.gz"
 
-get_src 3e6fe45f467d653870985cc52a1c2cf81a8a2c7a7bcf7ffcfedfd305a47a1eca \
-        "https://github.com/DataDog/dd-opentracing-cpp/archive/v$DATADOG_CPP_VERSION.tar.gz"
+get_src 202635ef6b9653ddbb8ca40089d0776544fe56fd86f944fcf04e7fdc5257710f \
+        "https://github.com/DataDog/dd-opentracing-cpp/archive/cgilmour/opentracing-1.6.0.tar.gz"
```

Comment:
let's use the existing `DATADOG_CPP_VERSION` to set the version instead of hard coding here